.travis.yml

```diff
@@ -14,7 +14,9 @@ matrix:
     fast_finish: true
     include:
         - python: 3.6
-          env: TOXENV=py3
+          env:
+              TOXENV=py3
+              NOSE_VERBOSE=2  # List all tests run by nose
```

Comment:
couldn't you set this env one level higher instead of explicitly in each test?

Reply:
I tried that initially, but it appears that the per-job `env` overrides the top-level `env`.

Reply:
i don't even know what to say to that

Reply:
"Approve"? :joy: